src/model/docker.ts

```diff
@@ -42,13 +44,18 @@ class Docker {
   static getBaseOsSpecificArguments(baseOs, workspace, unitySerial, runnerTemporaryPath, sshAgent): string {
     switch (baseOs) {
       case 'linux':
+        const github_home = join(runnerTemporaryPath, "_github_home");
+        existsSync(github_home) || mkdirSync(github_home);
+        const github_workflow = join(runnerTemporaryPath, "_github_workflow");
+        existsSync(github_workflow) || mkdirSync(github_workflow);
```

Comment:
I believe these directories should exist according to the spec. Could you explain a little bit about why this is needed?

Also prefer the more semantic `if(condition) // do something` over `||` which makes the code harder to read for new developers